src/Common/tests/System/IO/TempDirectory.cs

```diff
@@ -35,6 +35,8 @@ public void Dispose()
             DeleteDirectory();
         }
 
+        public string GenerateRandomFilePath() => IO.Path.Combine(Path, IO.Path.GetRandomFileName());
```

Comment:
Quite possibly, but not in this class (I believe we have one in one of the test base classes that uses this or one of the other temp directories). We need to get our temp directories and their usage cleaned up a bit. :)